flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/source/enumerator/TestContinuousSplitPlannerImpl.java

```diff
@@ -459,7 +459,7 @@ public void testIncrementalFromSnapshotTimestampWithInvalidIds() throws Exceptio
     AssertHelpers.assertThrows(
         "Should detect invalid starting snapshot timestamp",
         IllegalArgumentException.class,
-        "Cannot find a snapshot older than " + invalidSnapshotTimestampMsStr,
+        "Cannot find a snapshot after: ",
```

Comment:
nit: we can remove the variable def in line 446 above, as it is not used anymore after this change